### ISSUE TITLE
Add a review of STTN

### ISSUE THREAD

**Issue by jeffreycharters** [jeffreycharters/station#16]
Let people know how the self-taught tech network is helping you, what you think of it, etc.

**Comment by akebu6** [comment]
Hey Jeffrey, this issue is connected to being a part of the STTN correct?

**Comment by jeffreycharters** [comment]
Yes it is, if you are part of the discord server or familiar with anything that @joelpickin (on twitter) does, you can feel free to get in on this.

**Comment by akebu6** [comment]
I have seen what Joel does and I recently joined the server a few days ago

**Comment by AdityaShaw1** [comment]
can u pls assign me tjis issue
